pkg/reconciler/service/resources/configuration.go

```diff
@@ -38,6 +38,7 @@ func MakeConfiguration(service *v1alpha1.Service) (*v1alpha1.Configuration, erro
 			Labels: resources.UnionMaps(service.GetLabels(), map[string]string{
 				serving.ServiceLabelKey: service.Name,
 			}),
+			Annotations: service.GetAnnotations(),
```

Comment:
We need to inject the route label that's injected by the labeler, or the service controller is going to fight with it.

Reply:
As annotation?